RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate UserPreserencesView to Typescript

## Proposed changes



## Issue(s)	


## How to test or reproduce


- Go to `Profile`, then click on `Preferences` and should open a screen with one button `Notifications` and one Switch button `Enable message Parser`

## Screenshots

## Types of changes




## Checklist



## Further comments
